brain/python/numpyArrray.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
```

Comment:
Why have you moved this to a separate file? I prefer to keep it inside the only .cpp that uses it.